package.json

```diff
@@ -4,10 +4,13 @@
   "description": "basic routes",
   "main": "app.js",
   "scripts": {
-    "start": "concurrently \"npm run server\" \"react-scripts start\"",
-    "server": "nodemon server/app.js",
     "build": "react-scripts build",
-    "test": "eslint ./ && react-scripts test --env=jsdom"
+    "build-css": "node-sass-chokidar src/ -o src/devBuild",
+    "sass-lint": "sass-lint ./src --config .sass-lint.yml 'src/**/*.scss' --verbose --ignore 'node_modules/**/*'",
+    "server": "nodemon server/app.js",
+    "start": "concurrently --prefix \"[AAF]\" \"npm run watch-css\" \"npm run server\" \"react-scripts start\"",
+    "test": "eslint ./ && react-scripts test --env=jsdom",
```

Comment:
No Sass Linting? Is it time for a second travis test job?

Reply:
So if I have sass-lint in the test script, any results get's eaten up by the react-scripts test because it consumes the console control. 

Not sure how to get around it yet.